modules/ROOT/pages/prerequisites/index.adoc

```diff
@@ -265,11 +282,14 @@ a| {zfs_url}[ZFS]
 | Name
 | Default Block Size
 
-a| {nfs_url}[NFS]
+| {nfs_url}[NFS]
 | The block size depends on the `rsize` parameter in the mount options. Defaults to 4K, usually set to 32K.
 
-a| {cephfs_url}[CephFS]
-| CephFS does not have metadata by design, You have to use a POSIX-compliant filesystem (see list above) to store the metadata.
+| {cephfs_url}[CephFS]
+a| CephFS follows a different concept with regard to handling metadata, which impacts memory requirements. See the {cephfs_ram_url}[CephFS Hardware Recommendations] for more details.
+
+NOTE: Note that CephFS is not equal to Ceph-S3. +
+Ceph is an open source flexible distributed storage system (multi-server, multi-rack, multi-site) with an object storage layer. Ceph-S3 presents that object storage with an API that emulates the AWS' S3 API.
```

Comment:
note that you cannot access the same files in CephFS and Ceph-S3. Ceph allows exposing commodity hardware as either blockstorage (RBD), S3 or CephFS. IT is not possible to write a file via S3 and ther read it via CephFS.

Reply:
fixed